sdk/communication/Azure.Communication.Identity/api/Azure.Communication.Identity.net8.0.cs

```diff
@@ -74,8 +83,16 @@ public GetTokenForTeamsUserOptions(string teamsUserAadToken, string clientId, st
 }
 namespace Azure.Communication.Identity.Models
 {
-    public static partial class CommunicationIdentityModelFactory
+    public partial class CommunicationIdentityModelFactory
     {
+        public CommunicationIdentityModelFactory() { }
         public static Azure.Communication.Identity.CommunicationUserIdentifierAndToken CommunicationUserIdentifierAndToken(Azure.Communication.CommunicationUserIdentifier user, Azure.Core.AccessToken accessToken) { throw null; }
     }
+    public partial class CommunicationUserDetail
```

Comment:
Why is the class partial?

Reply:
From what I see every class in the API is added as partial (for example GetTokenForTeamsUserOptions)